src/var/getDefaultOptions.js

```diff
@@ -40,12 +40,36 @@ function($) {
             filters: {
                 tones: {
                     title: "Diversity",
-                    emoji: "santa runner surfer swimmer lifter ear nose point_up_2 point_down point_left point_right punch " +
-                    "wave ok_hand thumbsup thumbsdown clap open_hands boy girl man woman cop bride_with_veil person_with_blond_hair " +
-                    "man_with_gua_pi_mao man_with_turban older_man grandma baby construction_worker princess angel " +
-                    "information_desk_person guardsman dancer nail_care massage haircut muscle spy hand_splayed middle_finger " +
-                    "vulcan no_good ok_woman bow raising_hand raised_hands person_frowning person_with_pouting_face pray rowboat " +
-                    "bicyclist mountain_bicyclist walking bath metal point_up basketball_player fist raised_hand v writing_hand"
+                    emoji: "open_hands raised_hands clap pray thumbsup thumbsdown punch fist left_facing_fist right_facing_fist " + 
```

Comment:
This will broke support old versions of emojionearea

Reply:
I'm going to change and keep the same values as before.